src/controller/python/chip/ChipDeviceCtrl.py

```diff
@@ -906,7 +950,7 @@ async def FindOrEstablishPASESession(self, setupCode: str, nodeid: int, timeoutM
         if res.is_success:
             return DeviceProxyWrapper(returnDevice, DeviceProxyWrapper.DeviceProxyType.COMMISSIONEE, self._dmLib)
 
-    def GetConnectedDeviceSync(self, nodeid, allowPASE=True, timeoutMs: int = None):
+    def GetConnectedDeviceSync(self, nodeid, allowPASE=True, timeoutMs: int = None, payloadCapability: int = TransportPayloadCapability.MRP_PAYLOAD):
```

Comment:
This is matching existing functinality, I'm guessing, but do you want the default to be MRP or both?

Reply:
I guess existing devices don't support other capabilities? Or what would be the implications here?

There is `MRP_OR_TCP_PAYLOAD`, which I guess would choose MRP unless TCP is supported? :thinking:

Reply:
So, I have kept the default to match the existing functionality(as you noted). Making the default to MRPOrTCP would also require changes within the SDK APIs. If we choose to do that(which sounds perfectly reasonable), I would do that in a subsequent PR where it will contain that change throughout the SDK and the python scripts.